src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,376 @@
+| InstitutionAddress        | RECOMMENDED           | [string][]    | The address of the institution in charge of the equipment that produced the measurements.                                                                                                                 |
+| InstitutionDepartmentName | RECOMMENDED           | [string][]    | The department in the institution in charge of the equipment that produced the measurements.                                                                                                              |
+
+#### Image Acquisition
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Key name**               | **Requirement level** | **Data type**            | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| -------------------------- | --------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| PixelSize                  | REQUIRED              | [array][] of [numbers][] | A 2- or 3-number array of the physical size of a pixel, either`[PixelSizeX, PixelSizeY]` or `[PixelSizeX, PixelSizeY, PixelSizeZ]`, where X is the width, Y the height and Z the depth. If the file format is OME-TIFF, these values need to be consistent with `PhysicalSizeX`, `PhysicalSizeY` and `PhysicalSizeZ` OME metadata fields, after converting in `PixelSizeUnits` according to `PhysicalSizeXunit`, `PhysicalSizeYunit` and `PhysicalSizeZunit` OME fields. |
+| PixelSizeUnits             | REQUIRED              | [string][]               | Unit format of the specified PixelSize. MUST be micrometer (`"um"`).                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Was going to mention just that :)

Reply:
we should change that. no one is going to put an EM dataset in `um` it will likely be in `nm` hence we should remove the restriction of a single unit here.

Reply:
Thank you for this question!

We have basically two options here:
1)	Impose a unit for the field `PixelSize`
2)	Create a field `PixelSizeUnits` that enables the use of another unit.

We were planning on imposing `um` to the field `PixelSize` but chose in collaboration with @satra to go with option 2 for 2 reasons:
1)	For human-readability of the JSON. A user could see the unit without having to know what the imposed unit for `PixelSize` is.
2)	It left room for the spec to grow in the future with other units.

However, we were not planning on adding `nm` in this first version.
I don't have a strong opinion about this and would be comfortable with either imposing a unit in `PixelSize` or having a separate field. The latter may be more complicated for tools down the line to deal with though.
In any case, if we keep `PixelSizeUnits`, it should be restricted to a fixed set of possible units.